tests/test_wave_05.py

```diff
@@ -45,23 +42,16 @@ def test_get_goal(client, one_goal):
         }
     }
 
-
-@pytest.mark.skip(reason="test to be completed by student")
 def test_get_goal_not_found(client):
-    pass
     # Act
-    response = client.get("/goals/1")
+    response = client.get("/goals/999")  
     response_body = response.get_json()
 
-    raise Exception("Complete test")
     # Assert
-    # ---- Complete Test ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # ---- Complete Test ----
-
+    assert response.status_code == 404
+    assert "message" in response_body
+    assert response_body["message"] == "Goal 999 not found"
```

Comment:
Nice checking the status code! Since you are checking for your response body to have a key called "message" with a value of "Goal 999 not found",  I would be okay with removing the second `assert` statement.